content/tutorials/generic/micropython-installation/micropython-installation.md

```diff
@@ -0,0 +1,205 @@
+
+Download the .dfu file for the Portenta H7.
+
+### Finding dfu-util
+
+In this step you will be diving into the core files you downloaded in the first step. This process will be different for you depending on if you're on a Windows or MacOS computer.
+
+#### Windows
+
+The file you will need to access can be found in 
+```     
+C:\Users\{your username}\AppData\Local\Arduino15\staging\packages\dfu-util-0.10.0-arduino1-windows.tar.bz2\dfu-util-0.10.0-arduino1-windows.tar\windows\ 
+```
+and is named `dfu-util.exe`.
+
+One you've found the file, extract it from the .tar archive and copy it somewhere else, your desktop for example. 
+
+Open a command terminal and start typing in `start `, now drag and drop the dfu-util.exe file you've found into the terminal, and press enter.
+
+If successful, another command terminal window should open in which you are able to execute the command that will flash your board with the MicroPython firmware.
+
+Double tap the reset button on your board to put it in DFU mode, and execute the following command, but replace the path of the firmware file with the directory where you have placed the firmware you downloaded previously:
+
+```
+dfu-util -a 0 -d 0x2341:0x035b -D {firmware.dfu}
+```
+
+Again, don't disconnect or power off your board during this part of the process, watch the progress of flashing the firmware to your board. Once complete, you're all set and you're ready to start programming the board in MicroPython.
+
+Go to the Arduino Lab MicroPython IDE and press connect in the top left, choose the port, and code away!
+
+#### MacOS
+
+The file you will need to access can be found in 
+```
+Users/{your-user}/Library/Arduino15/packages/arduino/tools/dfu-util/0.10.0-arduino1/dfu-util
+```
+the file you are after is `dfu-util.exe`
+
+>**Note:** The `"Library"` directory is hidden, press `Shift + Command + . ` to reveal hidden directories. 
+
+Open a terminal window and drag and drop the dfu-util.exe file into the terminal window.
+
+Now, you should be able to execute the command that will flash your board with the MicroPython firmware.
+
+Double tap the reset button on your board to put it in DFU mode, and execute the following command, but replace the path of the firmware file with the directory where you have placed the firmware you downloaded previously:
+
+```
+dfu-util -a 0 -d 0x2341:0x035b -D {firmware.dfu}
+```
+
+Again, don't disconnect or power off your board during this part of the process, watch the progress of flashing the firmware to your board. Once complete, you're all set and you're ready to start programming the board in MicroPython.
+
+Go to the Arduino Lab MicroPython IDE and press connect in the top left, choose the port, and code away!
+
+### Flashing The Firmware
+
+With the tool installed execute the following command `dfu-util -a 0 -d 0x2341:0x035b -D firmware.dfu`
+
```

Comment:
@jacobhylen can you add a section that explains if the flashing is successful or not?